### PR TITLE
api: _* attributes preservation

### PR DESCRIPTION
**Pull request by hachreak**
Signed-off-by: Leonardo Rossi <leonardo.r@cern.ch>

### CODE REVIEW

**Comment by hachreak** [comment]
To resolve possible overwrite of '_*' fields in https://github.com/inveniosoftware/invenio-records-rest/blob/master/invenio_records_rest/views.py#L574-L575
WDYT?

-------------------------------------
cc @lnielsen @jirikuncar

**Comment by jirikuncar** [comment]
@hachreak please see https://github.com/inveniosoftware/invenio-deposit/pull/60

File: invenio_records/api.py [hunk 1]
```diff
@@ -91,6 +91,18 @@ def dumps(self, **kwargs):
...
 
+    def clear(self):
+        """Clear all except the field that start with `_`."""
+        for key in list(self.iterkeys()):
+            if not key.startswith('_'):
+                del self[key]
+
+    def update(self, *args, **kwargs):
+        """Update all except fields that start with `_`."""
+        for (key, value) in args[0].items():
```
Comment (jirikuncar) [inline invenio_records/api.py]: there are some other cases e.g. `record.update(foo='bar', baz='xyz')`

**Comment by hachreak** [comment]
could we use both?
The `status` is something specific for `deposit`. 
The behavior "preserving _* fields" could be considered a general behavior for all kind of records?

**Comment by jirikuncar** [comment]
It will prevent many low-level scripts and other dependencies to work correctly, hence I'm closing it for now.

**Pull request closed without merging**